common/flavorization/00_CTP.txt

```diff
@@ -7083,3 +7083,1773 @@ prince_female_tamil_kingdom = {
+	governments = { feudal_government clan_government }
+	top_liege = no
+	only_independent = yes
+}
+
+duke_feudalclan_male_bulgarian = {
+	type = character
+	gender = male
+	special = holder
+	tier = duchy
+	priority = 28
+	cultures = { bulgarian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+}
+
+duke_tribal_male_bulgarian = {
+	type = character
+	gender = male
+	special = holder
+	tier = duchy
+	priority = 28
+	cultures = { bulgarian }
+	governments = { tribal_government }
+	top_liege = no
+}
+
+duke_feudalclan_male_bulgarian_independent = {
+	type = character
+	gender = male
+	special = holder
+	tier = duchy
+	priority = 29
+	cultures = { bulgarian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+	only_independent = yes
+}
+
+duke_feudalclan_female_bulgarian = {
+	type = character
+	gender = male
+	special = holder
+	tier = duchy
+	priority = 28
+	cultures = { bulgarian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+	
+}
+
+duke_tribal_female_bulgarian = {
+	type = character
+	gender = female
+	special = holder
+	tier = duchy
+	priority = 28
+	cultures = { bulgarian }
+	governments = { tribal_government }
+	top_liege = no
+}
+
+duke_feudalclan_female_bulgarian_independent = {
+	type = character
+	gender = female
+	special = holder
+	tier = duchy
+	priority = 29
+	cultures = { bulgarian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+	only_independent = yes
+}
+
+duchy_feudalclan_bulgarian = {
+	type = title
+	tier = duchy
+	priority = 28
+	cultures = { bulgarian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+}
+
+duchy_tribal_bulgarian = {
+	type = title
+	tier = duchy
+	priority = 28
+	cultures = { bulgarian }
+	governments = { tribal_government }
+	top_liege = no
+}
+
+duchy_feudalclan_bulgarian_independent = {
+	type = title
+	tier = duchy
+	priority = 29
+	cultures = { bulgarian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+	only_independent = yes
+}
+
+
+king_feudalclan_male_bulgarian = {
+	type = character
+	gender = male
+	special = holder
+	tier = kingdom
+	priority = 48
+	cultures = { bulgarian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+}
+
+king_feudalclan_female_bulgarian = {
+	type = character
+	gender = male
+	special = holder
+	tier = kingdom
+	priority = 48
+	cultures = { bulgarian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+}
+
+kingdom_feudalclan_bulgarian = {
+	type = title
+	tier = kingdom
+	priority = 48
+	cultures = { bulgarian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+}
+
+emperor_feudalclan_male_bulgarian = {
+	type = character
+	gender = male
+	special = holder
+	tier = empire
+	priority = 103
+	cultures = { bulgarian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+}
+
+emperor_feudalclan_female_bulgarian = {
+	type = character
+	gender = male
```

Comment:
This line should say gender = female.

Reply:
Fixed all gender related errors. I'll resolve all of them